### PR TITLE
Add automatic status updates to UnityRoomUsers

### PR DESCRIPTION
**Pull request by kSlideHH**

### CODE REVIEW

File: g_python/hunitytools.py [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 from .gextension import Extension
 from .hmessage import HMessage, Direction
-from .hpacket import HPacket
-from .hunityparsers import HUnityEntity
+from .hunityparsers import HUnityEntity, HUnityStatus
 
 
 class UnityRoomUsers:
-    def __init__(self, ext: Extension, users_in_room=28, get_guest_room=385, user_logged_out=29):
+    def __init__(self, ext: Extension, users_in_room=28, get_guest_room=385, user_logged_out=29, status=34):
```
Comment (sirjonasxx) [inline g_python/hunitytools.py:10]: could you replace the header IDs with the packet names (does that work in g-python for unity? it should)

Comment (kSlideHH) [reply g_python/hunitytools.py:10]: I'll try if it works and then change it!

Comment (kSlideHH) [reply g_python/hunitytools.py:10]: Does not work unfortunately

Comment (sirjonasxx) [reply g_python/hunitytools.py:10]: You're aware thar the names changed right?

Comment (kSlideHH) [reply g_python/hunitytools.py:10]: yes, I took the names from messages.json
https://pastebin.com/mMTTkxWn

This is the version I currently have locally installed

Comment (sirjonasxx) [reply g_python/hunitytools.py:10]: do packet names work on the latest commit for you?

Comment (sirjonasxx) [reply g_python/hunitytools.py:10]: merging for now but making issue for this

**Pull request merged**